src/components/learnDetail/ScriptEdit.tsx

```diff
@@ -0,0 +1,123 @@
+import styled from 'styled-components';
+import { MutableRefObject } from 'react';
+import { COLOR } from '@src/styles/color';
+// import { FONT_STYLES } from '@src/styles/fontStyle';
+interface ScriptType {
+  id: number;
+  text: string;
+  startTime: number;
+  endTime: number;
+}
+
+interface ScriptEdit {
+  scripts: ScriptType[];
+  isHighlight: boolean;
+  isSpacing: boolean;
+}
+function ScriptEdit(props: ScriptEdit) {
```

Comment:
```suggestion
function ScriptEdit(props: ScriptEditProps) {
```
컨벤션 맞춰서 `__Props`로 작성합시다!